docker/nugus_sim/run.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+from typing import Tuple
+
+import ruamel.yaml
+
+BINARIES_DIR = "/home/nubots/NUbots/binaries"
+CONFIG_DIR = BINARIES_DIR + "/config"
+
+ENV_VARS = ["ROBOCUP_ROBOT_ID", "ROBOCUP_TEAM_COLOR", "ROBOCUP_SIMULATOR_ADDR"]
```

Comment:
Might as well make it immutable
```suggestion
ENV_VARS = ("ROBOCUP_ROBOT_ID", "ROBOCUP_TEAM_COLOR", "ROBOCUP_SIMULATOR_ADDR")
```